### DISCUSSION TITLE
Bass shake feature?

### DISCUSSION THREAD

**Discussion by green3554** [aleab/acav-we#8] [Ideas]
I love this visualizer, but I wish it had a bass shake feature. If you're not sure what I mean, I attached an example from another visualizer. 
(I hope I'm using github right I'm new to this)
https://user-images.githubusercontent.com/102485284/160297661-67f35e0c-54fe-4269-a684-e1a300015fc7.mp4

**Comment by aleab** [comment]
I actually have already implemented a bass shake feature, but only for the clock component.
It should be fairly easy to apply that to the visualizer as well.

EDIT: I take back what I said. Applying transformations to the 2D canvas where the visualizer is drawn is a non-trivial operation. It will take some time.

**Comment by aleab** [comment]
A simple version of this feature has been implemented for all visualizer types. I just pushed an update to Steam's Workshop.

**Reply by green3554** [reply]
Thank you!!!
